dask_cloudprovider/azure/azurevm.py

```diff
@@ -241,10 +246,44 @@ async def destroy_vm(self):
 class AzureVMScheduler(SchedulerMixin, AzureVM):
     """Scheduler running on an Azure VM."""
 
+    def __init__(self, *args, **kwargs):
+        kwargs.pop("preemptible", None)
+        super().__init__(*args, **kwargs)
+
+    async def start(self):
```

Comment:
Instead of overriding the `start` methods I would prefer to update the mixins themselves and expect `create_vm` to return a tuple. For clouds where this isn't being implemented yet they can just `return address, None`.